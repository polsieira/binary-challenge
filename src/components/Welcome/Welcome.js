import React from 'react';
import './Welcome.scss';

const Welcome = () => {

  return (
    <div className="Welcome">
      <h1 className="welcome-message">Don't limit yourself to just one planet</h1>
    </div>
  )


}

export default Welcome;