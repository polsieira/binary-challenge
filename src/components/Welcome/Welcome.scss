.Welcome {
  background-image: url('https://images.unsplash.com/photo-1518066000714-58c45f1a2c0a?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1650&q=80');
  background-size: 100% 100%;
  color: white;
  height: 300px;

  .welcome-message {
    font-family: 'Nova Square', cursive;
    margin: 0px;
    padding: 60px;
  }
}