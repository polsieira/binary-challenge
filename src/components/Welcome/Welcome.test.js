import Welcome from './Welcome'
import { shallow } from 'enzyme';
import React from 'react';


describe('Welcome', () => {

  let wrapper;

  describe('AccountManager component', () => {
    beforeEach(() => {
      wrapper = shallow(<Welcome />)
    })

    it('should match a snapshot', () => {
      expect(wrapper).toMatchSnapshot();
    });

  });

});

