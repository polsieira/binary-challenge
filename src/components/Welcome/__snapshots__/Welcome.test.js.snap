// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome AccountManager component should match a snapshot 1`] = `
<div
  className="Welcome"
>
  <h1
    className="welcome-message"
  >
    Don't limit yourself to just one planet
  </h1>
</div>
`;
